src/index.ts

```diff
@@ -271,17 +271,15 @@ class GetStarknetWallet implements IGetStarknetWallet {
 
         // lookup installed wallets
         const installed = Object.values(
-            ["starknet", "starknet_braavos", ...Object.keys(window)].reduce<{
-                [key: string]: IStarknetWindowObject;
-            }>((wallets, key) => {
+            Object.getOwnPropertyNames(window).reduce((wallets, key) => {
```

Comment:
pls declare `Record<string, IStarknetWindowObject>` here - directly on the `reduce` - instead of using `as` over line 282